src/plugin/hierarchical.ts

```diff
@@ -258,7 +258,8 @@ function handleClickEvents(
   _event: unknown,
   elem: { offset: number; index: number },
   offsetDelta: number,
-  inRange: (v: number) => boolean
+  inRange: (v: number) => boolean,
+  reverse:boolean = false
```

Comment:
please make sure to run `yarn fix` to check for linting and formatting fixes

Reply:
done